packages/uma-auth-client/src/NwcRequester.ts

```diff
@@ -0,0 +1,416 @@
+import {
```

Comment:
Might be good to include a comment at the top of this file indicating it was adapted from the alby sdk with a link to that file for reference (and to give them credit :-))

Reply:
ooh good idea!! ty